docker/sql-init/01_permissions.sql

```diff
@@ -0,0 +1,2 @@
+ALTER USER 'central_ledger'@'%' identified WITH mysql_native_password by 'password';
```

Comment:
Hi. What's the purpose of this?

Reply:
In the old onboarding doc for the `central-ledger`, we had some ungainly copying and pasting to get the docker containers running. (See [this](https://github.com/mojaloop/central-ledger/blob/dd86d346ed5900eed5c3c1ea48de7d63aedcbbe9/Onboarding.md#docker)).

This script is mounted inside the `mysql` container in docker and on first load will change the password of the central_ledger user.

Really, we're just making it easier to run the docker containers.

Reply:
Are you not able to set env vars for the MySQL docker container to set the DB, Pass, etc?

Reply:
Sure. I know that we already are setting env vars, so but I remember thinking there was a reason behind this script as well. I think it may have to do with the hostname of the central_ledger user.

I'll see if we can remove it altogether.

Reply:
Ah I remember now. 

We need this script because the version of MySQL we are using (8.x.x) doesn't support default authentication. When I removed the init script, despite having set the username and password in the environment variables, the `central-ledger` couldn't connect to the database.

Happy to discuss this further if needed - feel free to ping me on Slack.

Ref: https://github.com/mysqljs/mysql/issues/2013